independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json

```diff
@@ -0,0 +1,514 @@
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/LijnRichtingHits"
+            },
+            "examples": {
+              "application/json": {
+                "aantalHits": 6,
+                "lijnrichtingen": [
+                  {
+                    "lijnNummerPubliek": "335",
+                    "entiteitnummer": "3",
+                    "lijnnummer": "335",
+                    "richting": "HEEN",
+                    "omschrijving": "Leuven - Aarschot",
+                    "bestemming": "LEUVEN GASTHUISB.",
+                    "kleurVoorGrond": "#FFFFFF",
+                    "kleurAchterGrond": "#991199",
+                    "kleurAchterGrondRand": "#991199",
+                    "links": [
+                      {
+                        "rel": "string",
+                        "url": "string"
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          },
+          "500": {
+            "description": "service of achterliggende systemen niet beschikbaar"
+          }
+        }
+      }
+    },
+    "/zoek/locaties/{zoekArgument}": {
+      "get": {
+        "summary": "zoekLocaties",
+        "description": "Search locations (x,y) by user adress input. Typical for input on a routeplanner (eg kerkstraat 14 Hasselt or Achter De Kazerne ) . Limitied Geocoding service that retuns stops, adresses or POIs and its XY coordinates",
+        "operationId": "ZoekLocaties",
+        "produces": [
+          "application/json;charset=UTF-8"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "zoekArgument",
+            "description": "het zoekargument",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "huidigePositie",
+            "description": "momenteel niet ondersteund voor lokaties",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "startIndex",
+            "description": "Format - int32. de start positie in de lijst van gevonden lokaties",
+            "type": "integer",
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "maxAantalHits",
+            "description": "Format - int32. het maximaal aantal lokaties aangeleverd door deze service ",
+            "type": "integer",
+            "default": 10
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/LocatiesHits"
+            },
+            "examples": {
+              "application/json;charset=UTF-8": {
+                "aantalHits": 51,
+                "locaties": [
+                  {
+                    "type": "adres",
```

Comment:
@Lschockaert can you please explain what you mean by "adres", is it "address"?
if yes may I ask you please to change it in all the swagger file